File: server/test/testserver/testserver/message/Protocols/login_req.cs
using messages;

namespace messages.Protocols
{
  public class login_req : MsgBase
  {
      public login_req()
      {
          CodeId = MsgCodeId.login_req;
      }
      public string tel;    ////手机号
      public string pass;    ////密码

	}
}

File: client/butterfly/app/src/main/java/pf/com/butterfly/message/Protocols/login_req.java
package pf.com.butterfly.message.Protocols;
import android.util.Log;
import org.json.JSONArray;
import org.json.JSONObject;
import pf.com.butterfly.message.MsgBase;
import pf.com.butterfly.message.MsgCodeId;
  public class login_req extends MsgBase
  {
      public login_req()
      {
          CodeId = MsgCodeId.login_req;
      }
      public String tel;    ////手机号
      public String pass;    ////密码

}
